Common/include/CConfig.hpp

```diff
@@ -6073,6 +6091,12 @@ class CConfig {
     MarkerRotation_Rate[3 * iMarkerMoving + iDim] = val;
   }
 
+  /*!
+   * \brief Get the final translation velocity of the mesh in the y-direction
+   * \return Velocity of the mesh in the y-direction
+   */
+  su2double GetFinalTranslation_Rate_Y() const { return FinalTranslation_Rate_Y;}
+
```

Comment:
Can this be removed now? And maybe some of the older functions to set/get Z rotation?